http/1.7/modules/ROOT/pages/http-request-ref.adoc

```diff
@@ -153,7 +153,7 @@ You can use DataWeave expressions to add headers as well:
 
 . In the HTTP *Request* operation configuration window, select the *Headers* tab.
 . Click *fx* to switch to literal mode.
-. In the box located in the right section of the *fx* button, add the DataWeave expression `[{'HeaderName1' : 'HeaderValue1'}]`.
+. In the box located in the right section of the *fx* button, add the DataWeave expression `{'HeaderName1' : 'HeaderValue1'}`.
```

Comment:
Is there a way you can get around using the term "right"? This term is considered non-accessible, but I know we can't always get around that.